src/crashsim.py

```diff
@@ -66,23 +67,28 @@ def main():
   logging.basicConfig(level=args.verbosity)
 
   # creating the test
-  proc_create = subprocess.Popen(["rrtest", "create", "--name", args.cmd,
-      "--command", args.command, "-f", args.force])
-  proc_create.wait()
+  print("-----------------------")
+  create_test(args.cmd, args.command, args.force, args.verbosity)
+  # proc_create = subprocess.Popen(["rrtest", "create", "--name", args.cmd,
```

Comment:
It's best to go ahead and delete code rather than comment it out.  We can always retrieve it from history if we need to.